tasks/offlinePackagingTasks.ts

```diff
@@ -73,22 +73,21 @@ gulp.task('installDependencies', async () => {
     }
 });
 
-// Install Tasks
-async function installRoslyn(packageJSON: any, platformInfo?: PlatformInformation) {
-    const roslynVersion = packageJSON.defaults.roslyn;
-    const packagePath = await acquireNugetPackage('Microsoft.CodeAnalysis.LanguageServer', roslynVersion);
+gulp.task('fetchAllRoslynPlatformPackages', async () => {
```

Comment:
Can you update the contributing docs to mention that you should run this task to update roslyn instead of the 'installDependencies' task?

Reply:
Ah yes, right.

Reply:
Not only did I update the documentation, I also updated this to run installDependencies after. And renamed it to "updateRoslynVersion" which is just a better name, honestly.